db/sqldb/owner.go

```diff
@@ -142,10 +142,13 @@ func (d *DB) DeleteOwner(tx db.Tx, oID pacta.OwnerID) ([]pacta.BlobURI, error) {
 			return fmt.Errorf("getting portfolio groups for owner: %w", err)
 		}
 		for _, pgroup := range pgroups {
-			err := d.DeletePortfolioGroup(tx, pgroup.ID)
+			pgBuris, err := d.DeletePortfolioGroup(tx, pgroup.ID)
```

Comment:
Not sure if I called this out when this function was added, but does it make sense to delete "all the things" when deleting an owner? I.e if a user says "delete this owning user/initiative", do we want to recursively delete portfolio groups and everything beneath them? Or should we just fail and say "sorry this owner owns portfolio groups that need to be moved first"?

I ask because, while I'm glad we aren't using `CASCADE`, these functions are the equivalent of a `CASCADE` relationship and so we should make sure that the user-intention matches up with the DB intention